.circleci/config.yml

```diff
@@ -146,14 +168,40 @@ jobs:
             - kube-compose-controller-coverage.tar
             - kube-compose-installer.tar
             - kube-compose-e2e-tests.tar
+  e2e-kind:
+    docker: *public-golang 
+    working_directory: /root/src/github.com/docker/compose-on-kubernetes
+    environment:
+      GOPATH: "/root"
+    steps:
+      - attach_workspace:
+          at: /root
+      - setup_remote_docker:
+          version: 18.06.0-ce
+      - run:
+          name: "Prepare Kind and run E2E tests"
+          command: |
+            go get sigs.k8s.io/kind            
```

Comment:
We should change this to a fixed release when we can.

Reply:
we can always `git clone` manually, checkout a specific commit, and go install. Will do that.